src/main/java/org/kainos/ea/api/DeliveryEmployeeService.java

```diff
@@ -0,0 +1,51 @@
+package org.kainos.ea.api;
+
+import org.kainos.ea.cli.DeliveryEmployee;
+import org.kainos.ea.cli.DeliveryEmployeeRequest;
+import org.kainos.ea.client.FailedToCreateDeliveryEmployeeException;
+import org.kainos.ea.client.FailedToGetDeliveryEmployeesException;
+import org.kainos.ea.client.InvalidDeliveryEmployeeException;
+import org.kainos.ea.core.DeliveryEmployeeValidator;
+import org.kainos.ea.db.DeliveryEmployeeDao;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class DeliveryEmployeeService {
+
+    private final DeliveryEmployeeDao deliveryEmployeeDao = new DeliveryEmployeeDao();
```

Comment:
Is there a way you can improve this to follow SOLID principles. For example you could use dependency injection here